common/darwin/Classes/AudioUtils.m

```diff
@@ -67,26 +67,14 @@ + (void)setSpeakerphoneOn:(BOOL)enable {
     [session lockForConfiguration];
     NSError *error = nil;
     if(!enable) {
-        [session setMode:config.mode error:&error];
-        BOOL success = [session setCategory:config.category
-                                withOptions:AVAudioSessionCategoryOptionAllowAirPlay|AVAudioSessionCategoryOptionAllowBluetoothA2DP|AVAudioSessionCategoryOptionAllowBluetooth
-                                      error:&error];
```

Comment:
These calls to `setCategory` would sometimes fail. [`AVAudioSessionCategoryOptionAllowAirPlay`](https://developer.apple.com/documentation/avfaudio/avaudiosessioncategoryoptions/avaudiosessioncategoryoptionallowairplay) can only be set if the category is [`AVAudioSessionCategoryPlayAndRecord`](https://developer.apple.com/documentation/avfaudio/avaudiosessioncategoryplayandrecord), and there are times when `config.category` is something different.